src/main/resources/schemas/nhinc/common/PropertyAccess.xsd

```diff
@@ -134,6 +135,38 @@
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="DeletePropertyFileResponse" type="tns:DeletePropertyFileResponseType" />
-    
+    <xsd:complexType name="AddPropertyRequestType" >
+        <xsd:sequence>
+            <xsd:element name="file" type="xsd:string" />
```

Comment:
Both AddPropertyRequestType and UpdatePropertyRequestType can use the same object